### PR TITLE
Provided additional functionality around chatgpt requests

### PR DESCRIPTION
**Pull request by DDoyle1066**
## What I did
- Added function to generate messages in the format needed to send a request to OpenAI API
- Added system instructions that have been effective so far in generating good summaries
- Brought function in from OpenAI's docs to precalculate number of tokens needed for the request

## Why I did it
- System instructions are needed to tell chatgpt what to focus on when summarizing and tagging bills
- Messages need to be formatted a certain way for the API
- Counting tokens will be important to understand cost per API request

## Issue #s Addressed
Building blocks for #7 
Also building block for #4
Does not close any issue

## How to test it
generate_messages() function can be tested by inputting request_types and text input and observing that it pulls the right function in the message generation process.
- get_system_instructions() can be tested by copying the output into the "Custom Instructions" section of ChatGPT (accessible from the popup menu generated by clicking on your name in the bottom right) and observing ChatGPT's performance on bills that are copied into the chat.

Screenshots below of system instructions:
image
The following shows the ability to detect large sums of money per the system instructions after copying in the first draft of SB106 as Bill 1 and the same draft as Bill 2 with the additional line added of: ALLOCATE $10B TO PROJECT STARDUST
image

### CODE REVIEW

**Comment by maxachis** [comment]
Connected to PR #21

File: src/summarizer/ChatGPT_prompts.py
Comment (maxachis) [inline src/summarizer/ChatGPT_prompts.py:1]: One risk we might have is that ChatGPT may have a response like "Sure, I'll summarize that for you", and then provide the summary. Not debilitating, but would be distracting. We'll need to test to see if that occurs.

File: src/summarizer/ChatGPT_prompts.py [hunk 1]
```diff
@@ -1,14 +1,123 @@
+import openai
+import tiktoken
+
+
+def get_system_instructions():
+    return "Limit responses to 200 words." \
```
Comment (maxachis) [inline src/summarizer/ChatGPT_prompts.py:6]: What's our rationale for 200 words as opposed to some other limitation?

Comment (DDoyle1066) [reply src/summarizer/ChatGPT_prompts.py:6]: The 200 words was not a hard and fast rule. 100 words seemed too terse. No limit led to very long summaries which I think would only get worse for larger bills.

File: src/summarizer/ChatGPT_prompts.py
Comment (DDoyle1066) [reply src/summarizer/ChatGPT_prompts.py:1]: I don't recall that happening but I will keep my eye out as I test

**Pull request merged**